client/src/NewVideoForm.jsx

```diff
@@ -0,0 +1,44 @@
+import "./NewVideoForm.css";
+const NewVideoForm = () => {
+	const handleSubmit = (e) => {
+		e.preventDefault();
+
+		fetch("/api/videos", {
+			method: "POST",
+			headers: {
+				Accept: "application/json",
+				"Content-Type": "application/json",
+			},
+			body: JSON.stringify({
+				title: e.target.title.value,
```

Comment:
This is a perfectly acceptable way of handling inputs, especially for a small form, however it's  worth being aware that there's a different (not necessarily better, it just serves a different purpose) way of doing this that you may need someday

https://react.dev/reference/react-dom/components/input#controlling-an-input-with-a-state-variable